README.md

```diff
@@ -55,3 +55,12 @@ Click on the canvas to enter fullscreen+pointerlock.  Plug in a VR headset, fire
 Move yor head around, turn from side to side, roll, look up and down, and move around within the front facing position sensor's frustum.  Click this screenshot for video!
 
 [![](screenshots/13.png)](https://vimeo.com/136548758)
+
+ ##Run
+ Install nodejs and npm
+ pm install -g  bower
```

Comment:
you can remove this whole section since it's documented in the `README` now:

https://github.com/nickdesaulniers/joshVR/blob/master/README.md#local-development